### PR TITLE
websocketd 0.4.1

### PR DESCRIPTION
**Pull request by hyuraku**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----
Remove gopath as formula supports Go modules.

Relates to #47627.

### CODE REVIEW

File: Formula/websocketd.rb [hunk 1]
```diff
@@ -16,15 +16,9 @@ class Websocketd < Formula
...
-    ENV["GOPATH"] = buildpath
-
-    src = buildpath/"src/github.com/joewalnes/websocketd"
-    src.install buildpath.children
-    src.cd do
-      system "go", "build", "-ldflags", "-X main.version=#{version}", "-o", bin/"websocketd"
-      man1.install "release/websocketd.man" => "websocketd.1"
-      prefix.install_metafiles
-    end
+    system "go", "build", "-ldflags", "-X main.version=#{version}", "-o", bin/"websocketd"
```
Comment (carlocab) [inline Formula/websocketd.rb:19]:
```suggestion
    system "go", "build", "-ldflags", "-X main.version=#{version}", *std_go_args
```

File: Formula/websocketd.rb [hunk 2]
```diff
@@ -16,15 +16,9 @@ class Websocketd < Formula
...
+    man1.install "release/websocketd.man" => "websocketd.1"
+    prefix.install_metafiles
```
Comment (carlocab) [inline Formula/websocketd.rb:21]:
```suggestion
```

**Review by carlocab — COMMENTED** [review]
Thanks, @hyuraku.

**Comment by hyuraku** [comment]
@carlocab 
https://github.com/joewalnes/websocketd/releases/tag/v0.3.1
`websocketd` was version 0.3.1 and GitHub pre-release, so tests failed.
Should I downgrade `websocketd`?

**Comment by carlocab** [comment]
Yes, please. Add `version_scheme 1` so that audit won't complain about the decrease in version.

**Comment by chenrui333** [comment]
let me also file an issue in the upstream to request a new release.

**Comment by chenrui333** [comment]
The upstream is still using pre-release tag, so it would fail again.

**Comment by chenrui333** [comment]
requeue the build as 0.4.1 is the latest release.

**Review by carlocab — APPROVED** [review]

**Pull request closed without merging**